src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const errorMessage = () => {
+  const message = document.createElement('div');
+
+  message.classList.add('message', 'error-message');
+  message.innerHTML = 'Promise was rejected!';
+  document.body.append(message);
+};
+
+const succesMessage = () => {
```

Comment:
Same here. Also, functions for showing success and error messages are almost the same. It would be great to combine them into one function